dbms/programs/server/Server.cpp

```diff
@@ -556,8 +556,26 @@ int Server::main(const std::vector<std::string> & /*args*/)
     try
     {
         loadMetadataSystem(*global_context);
-        /// After attaching system databases we can initialize system log.
-        global_context->initializeSystemLogs();
+
+        if (!config().getBool("system_tables_lazy_load", true))
+        {
+            try
+            {
+                /// If the flag system_tables_lazy_load is false, after attaching system databases
+                ///  we can try to initialize system logs non-lazily.
+                global_context->createSystemLogs();
+            }
+            catch (...)
+            {
+                LOG_ERROR(log, "Caught exception while loading system tables.");
+                throw;
+            }
+        } else
```

Comment:
coding style issue, correct is

```
if (foo)
{
}
else
{
}
```

Reply:
See also https://clickhouse-test-reports.s3.yandex.net/9642/e450fb1e800211448b5a635615eecdf298a45a21/style_check.html